great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -1637,6 +1774,83 @@ def _sqlalchemy_column_map_condition_values(
     ]
 
 
+def _sqlalchemy_column_pair_map_condition_values(
+    cls,
+    execution_engine: SqlAlchemyExecutionEngine,
+    metric_domain_kwargs: Dict,
+    metric_value_kwargs: Dict,
+    metrics: Dict[str, Any],
+    **kwargs,
+):
+    """Return values from the specified domain that match the map-style metric in the metrics dictionary."""
+    (
+        boolean_mapped_unexpected_values,
+        compute_domain_kwargs,
+        accessor_domain_kwargs,
+    ) = metrics["unexpected_condition"]
+
```

Comment:
@NathanFarmer Minor: Let us delete this line (to keep up the existing style).  Thanks!